gui/app/locales/fr.json

```diff
@@ -42,6 +42,9 @@
   "Error {0} names cannot end with character {1}": "un nom de {0} ne peut pas se terminer par un caractère {1}",
   "Error {0} names cannot include characters {1}": "un nom de {0} ne peut pas inclure de caractères {1}",
   "Error the “{0}” name is reserved": "le nom « {0} » est réservé",
+  "Error it exceeds the path size limit": "il dépasse la limite de taille pour les chemins",
+  "Error it exceeds the name size limit": "il dépasse la limite de taille pour les noms",
+  "Error it exceeds the folder name size limit": "il dépasse la limite de taille pour les noms de dossiers",
```

Comment:
These lines should be added to `gui/app/locales/en.json` too, no? (and `tx push -s` to make them available to translate in transifex)